src/components/CurrentRefinements.vue

```diff
@@ -5,26 +5,46 @@
   >
     <slot
       :refine="state.refine"
-      :items="refinements"
+      :items="state.items"
       :createURL="state.createURL"
     >
       <ul :class="suit('list')">
         <li
-          v-for="item in refinements"
-          :key="item.key"
+          v-for="item in state.items"
+          :key="item.attribute"
+          :class="suit('item')"
         >
           <slot
             name="item"
-            :refine="state.refine"
+            :refine="item.refine"
             :item="item"
-            :createURL="() => state.createURL(item.value)"
+            :createURL="state.createURL"
           >
-            <span :class="suit('item')">
-              <span :class="suit('label')">{{ item.attribute | capitalize }}: {{ item.label }}</span>
-              <button
-                :class="suit('delete')"
-                @click="state.refine(item.value)"
-              >✕</button>
+            <span :class="suit('label')">{{ item.attribute | capitalize }}: </span>
```

Comment:
If you rely on [InstantSearch.js 3.1.0](https://unpkg.com/instantsearch.js@3.1.0/), you can now use `item.label`.